assets/src/application/DomDataTypes.ts

```diff
@@ -0,0 +1,63 @@
+import type { CurrencyProps, CurrentUserProps, GeneralSettings, SiteUrlProps } from './valueObjects/config/types';
+
+export interface EventEspressoDomData {
+	api: ApiDomData;
+	config: ConfigDomData;
+	readonly domain: string;
+	eei18n: EspressoTranslations;
+	i18n: I18nData;
+}
+
+export type ApiDomData = {
+	graphqlEndpoint: string;
+	restApiBaseUrl: string;
+	restApiCollectionEndpoints: { [key: string]: string };
+	restApiNonce: string;
+	restApiPrimaryKeys: { [key: string]: string | [string] };
```

Comment:
```suggestion
	restApiCollectionEndpoints: AnyObject;
	restApiNonce: string;
	restApiPrimaryKeys: AnyObject<string | [string]>;
```

`AnyObject` comes from `@appServices/utils`

Reply:
done thnx